app-ios/Plugins/SwiftGenPlugin/Plugin.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import PackagePlugin
+
+@main
+struct SwiftGenPlugins: BuildToolPlugin {
+    func createBuildCommands(context: PluginContext, target: Target) async throws -> [Command] {
+        let configurations: [Path] = [context.package.directory, target.directory]
+          .map { $0.appending("swiftgen.yml") }
```

Comment:
Hey 👋 Thanks for this temporary workaround, it worked great in Xcode 13. However, in Xcode 14 we experience issues, because it is looking for the `swiftgen.yml` config file in both places and if one of them doesn't exist, the whole command fails. 

The solution is to add `.filter { FileManager.default.fileExists(atPath: $0.string) }`.
Thanks to @Carrione for this idea! 🚀

Reply:
@pchmelar NP :)